main/src/com/google/refine/exporters/CustomizableTabularExporterUtilities.java

```diff
@@ -390,9 +390,9 @@ CellData format(Project project, Column column, Cell cell) {
                     return new CellData(column.getName(), value, text, link);
                 }
             }else {//added for sql exporter
-            
-                if(includeNullFieldValue) {
-                    return new CellData(column.getName(), "", "", "");
+            	 includeNullFieldValue = true;
```

Comment:
A more compact way to do this would be to just remove the `if` statement, but I'm concerned about the comment directly above which seems to imply that other exporters might have a dependency on this.